signed/verify_test.go

```diff
@@ -132,7 +132,7 @@ func Test(t *testing.T) {
 			role: "root",
 			name: "expired",
 			exp:  &expiredTime,
-			err:  ErrExpired{"root", expiredTime.Format("2006-01-02 15:04:05 MST")},
+			err:  ErrExpired{"root", expiredTime.Format("Mon Jan 2 15:04:05 MST 2006")},
```

Comment:
This is just meant to be human readable (it gets output somewhere I think, I know I've seen it in the terminal). The default format includes milliseconds.